### ISSUE TITLE
Adding a My team section for 4 to 5 team cards

### ISSUE THREAD

**Issue by pulkithanda** [pulkithanda/voyager-hf#7]
<h2 align="center">Issue picture</h2>
<div align="center">
<p> Adding a My team section for 4 to 5 team cards (for now use random names)</p>
<img src="https://cdn.discordapp.com/attachments/787749276956491806/887762968014622740/unknown.png" height="150px" width="300px">
<img src="https://cdn.discordapp.com/attachments/787749276956491806/887763208599896094/unknown.png" height="150px" width="300px">
<p>You may use the following link for your reference:</p>
<a href="https://freefrontend.com/css-cards/"> https://freefrontend.com/css-cards/ </a> </p><br>
<p> For further Issues contact below or use my discord to contact me <i>Alphabet# 0352</i> </p>
</div>

**Comment by artimanputra** [comment]
I want to work on this section

**Comment by artimanputra** [comment]
I would like to work on this issue.

**Comment by artimanputra** [comment]
I want to work on this issue.

**Comment by pulkithanda** [comment]
Hey there! Sure go ahead, Assigned the issue to you

**Comment by artimanputra** [comment]
I made this team section. Let me know if any changes can be done. Then I will send a pull request. (Please ignore the watermark in the video it is due to software)[https://drive.google.com/file/d/12gi2bywM5BWZxPuBhs2RHSOwpViAMkQz/view?usp=sharing](url)

**Comment by pulkithanda** [comment]
Please add a heading tag below the about us section and then add this team section. A little spacing btw the buttons would do the trick. Also If you could try to make it responsive it would be great. Rest everything is nice! One more thing, If you'd submit a PR now and I accept it, It won't be counted in hacktoberfest( If you're participating for that). Your PR will only be counted for hacktoberfest if you submit it in October. So do let me know below.

**Comment by artimanputra** [comment]
Thanks for all the suggestions. I will make it responsive. Yeah, I am going to participate in hacktoberfest, sure I will submit my PR in October.

**Comment by pulkithanda** [comment]
Yea great just wait for October then! Meanwhile, you can select other issues from other ppl even ill be posting many more. Good luck!

**Comment by vampnik** [comment]
Hey
Can I work on this our team page?

**Comment by prachi0011** [comment]
Hey, I want to work on this issue. Actually I have done this task in project as well.

**Comment by pulkithanda** [comment]
@vampnik Yeah sure gimme some time exams going on. will tell you in sometime

**Comment by vampnik** [comment]
@pulkithanda Sure no problem
